Controls/LoadablePanel/src/DotVVM.Contrib/LoadablePanel.cs

```diff
@@ -72,15 +72,28 @@ public bool HideUntilLoaded
             DotvvmProperty.Register<bool, LoadablePanel>(t => t.HideUntilLoaded);
 
         /// <summary>
-        /// Gets or sets a collection of values of all loadable panels which are currently loading.
+        /// Gets or sets a collection of element ids of all loadable panels which are currently in the loading state.
         /// </summary>
-        public IEnumerable LoadingItems
+        public IEnumerable LoadingElementIds
         {
-            get { return (IEnumerable)GetValue(LoadingItemsProperty); }
-            set { SetValue(LoadingItemsProperty, value); }
+            get { return (IEnumerable)GetValue(LoadingElementIdsProperty); }
+            set { SetValue(LoadingElementIdsProperty, value); }
         }
-        public static readonly DotvvmProperty LoadingItemsProperty =
-            DotvvmProperty.Register<IEnumerable, LoadablePanel>(t => t.LoadingItems, null);
+        public static readonly DotvvmProperty LoadingElementIdsProperty =
+            DotvvmProperty.Register<IEnumerable, LoadablePanel>(t => t.LoadingElementIds, null);
+
+        /// <summary>
+        /// Gets or sets a key that identtifies the data displayed in the loadable panel. When the key value changes the panel is reloaded.
+        /// This property needs to be bound if the loadable panel is in a GridView Repeater or other client-updated controls.
+        /// </summary>
+        [MarkupOptions(AllowBinding = true, AllowHardCodedValue = false)]
+        public string Key
```

Comment:
I'd prefer using some more descriptive name than just `Key` - key to what? 
What about `ContentKey`, `DataKey` or something like that?

Reply:
I renamed the property to `ContentReloadBinding` this should be more acurate